docs/examples/spiking-mnist.ipynb

```diff
@@ -159,13 +159,10 @@
     "\n",
     "# when testing our network with spiking neurons we will need to run it \n",
     "# over time, so we repeat the input/target data for a number of \n",
-    "# timesteps. we're also going to reduce the number of test images, just \n",
-    "# to speed up this example.\n",
+    "# timesteps.\n",
```

Comment:
I think the changes in this file probably belong in an earlier commit?

Reply:
I put it in this commit because this is a change that is made possible by the change to running the docs on the GPU server.

Reply:
Ah gotcha, thought it was more of a bugfix as I didn't read the diff.